### ISSUE TITLE
Feature Request for V2.0

### ISSUE THREAD

**Issue by trummerschlunk** [trummerschlunk/master_me#112]
- [ ] true peak limiting

**Comment by magnetophon** [comment]
As far as I know, you cannot do true peak limiting without latency, and I thought one of the main goals was to have 0 latency.
I am very interested in making a different plugin with oversampling and lookahead, however.

**Comment by danimo** [comment]
@magnetophon That's correct. Most commericial plugins have a tradeoff mode where they try to get close to TP or achiving TP by trading in zero-latency for some latency. Depending on the scenario, either might be acceptable, but a (close to TP) cutoff is a must because you need to to have a good idea about your planned headroom for further loss of headroom during lossy audio file compression (which can be >1dB, potentially causing clipping with tight peak values).
